### ISSUE TITLE
connectors don't actually connect

### ISSUE THREAD

**Issue by pinesh** [tsunamayo/Starship-EVO#2362]
### Short Description of the issue:
Connecting two entities with connectors doesn't actually connect them, colliding with them will cause them to fly apart
...

### Steps to reproduce the issue:
build two entities, and connect them, then try to ram one. 
...

### Environment
- Starship EVO version: 20w23d

- Graphics Card: 
Vega 56
### Additional information:
https://giphy.com/gifs/RjwL8qVRPbJOCBxyVS/html5
https://gph.is/g/Z5B2eKo

_(Screenshots, Videos, Error logs, Related issues)_

**Comment by pinesh** [comment]
links are a GIF to a demonstration

**Comment by tsunamayo** [comment]
Yes this is not implemented for ship docking - it is more a station thing for now but it was next on my list. Thanks

**Comment by pinesh** [comment]
Ah ok, i was trying to make a station at the time from starter blocks, should i have done this differently to have it stay together?

**Comment by tsunamayo** [comment]
No thats fine, I will just add a few block to make it a proper station, like an anchor block (so it dont move), and a broadcast antenna so see it from afar.

**Comment by pinesh** [comment]
fantastique! Expect a modular 600m behemouth in the coming weeks. 

![image](https://user-images.githubusercontent.com/14081996/83982995-afdea980-a8df-11ea-9604-ebf12919b186.png)

**Comment by Womble-TC** [comment]
Would be cool if the anchor block looked something like this:
![image](https://user-images.githubusercontent.com/29961381/83983081-6e6fed80-a923-11ea-969a-3d4455e6d922.png)
This is a totally original and completely not from the game files (100% true)

**Comment by Garrett-C** [comment]
> Would be cool if the anchor block looked something like this:
> ![image](https://user-images.githubusercontent.com/29961381/83983081-6e6fed80-a923-11ea-969a-3d4455e6d922.png)
> This is a totally original and definitely not from the game files (100% true)

Nice concept I like it. You are a master of creating original content womble. I love how well you made it fit the game's style.

**Comment by tsunamayo** [comment]
@Womble-TC okay how did you saw this lol?

**Comment by tsunamayo** [comment]
This was going to be the mag-lock block fyi, station anchor is gonna be more massive, but I will need to change the icon.

**Comment by Garrett-C** [comment]
> This was going to be the mag-lock block fyi, station anchor is gonna be more massive, but I will need to change the icon.

Oh interesting. Do you know if you are going to have it need to be on both ships to lock or only one? And will there be a brick option for fighters/landing gear?

**Comment by tsunamayo** [comment]
So only on the parent ship, and I want a ship-to-ship command link so you can toggle in on the children ship. This is 0.5m block.

**Comment by Garrett-C** [comment]
> So only on the parent ship, and I want a ship-to-ship command link so you can toggle in on the children ship. This is 0.5m block.

Sounds perfect so far.

**Comment by Womble-TC** [comment]
@tsunamayo Someone on the discord managed to brute-force it to spawn by changing the values of a blueprint for just a single block in a text editor.

**Comment by tsunamayo** [comment]
lol I would like to know how he did that, stuff is zipped then not really human readable.
@pinesh it looks great but it has no chance of ending in the game, as it would kill performance too much I think. Your ship looks great too, if you can show it to us a bit more!

**Comment by Sullos** [comment]
I can't remember the process for sure. But from memory, the player made a blueprint of a single block, exported it, opened the blueprint in sublime text(just a fancy text editor some of us use), and viewed it as hexadecimal code, then proceeded to change one value at a time until they figured out which value corresponded to block ID until they figured it out. Not really something players can do on a massive scale, but easy enough to do 1 block at a time with some brute force. And also not really something that can be fixed, nor worthwhile fixing/changing.

**Comment by tsunamayo** [comment]
@Sullos @Womble-TC pro-tip for that poor guy, you can go in the save folder and check the list of all block ID inside bricks.csv. When I have time I will export a nice description + stat + high res icon for wiki or else.
